alcs-api/src/application/application-meeting/application-meeting.controller.spec.ts

```diff
@@ -149,26 +148,4 @@ describe('ApplicationMeetingController', () => {
       reportEndDate: meetingToUpdate.reportEndDate,
     });
   });
-
-  it('should fail create meeting if meeting type does not exist', async () => {
```

Comment:
Why is this test no longer relevant?

Reply:
Before my change, this was using .find() meaning we had our own check if it was null and throw our own error. Now that its using findOrFail, it will automatically throw an exception if it does not exist. So since the logic is now part of typeorm (and would need to be mocked in), we don't need a unit test for it.